typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/presentation.ts

```diff
@@ -48,21 +54,15 @@ export function printWorkflowXml(workflow: WorkflowDescriptor, context: FileTran
 		stringBuilder.append(`<description><![CDATA[${workflow.description}]]></description>`).appendLine();
 	}
 
-	let graph: Graph;
-
-	try {
-		graph = getGraph(workflow);
-	} catch (e) {
-		throw new Error(`Error while building graph for workflow "${workflow.name}". Please check the workflow for any missing target items.
-                        Original Error: ${e.message}`);
-	}
+	const graph: Graph = getGraph(workflow);
 
-	stringBuilder.append(`<position x="${graph.getNodeSpacing() / 2}" y="${graph.getHeight() / 2}" />`).appendLine();
+	const startNode = graph.getNode(Graph.START);
```

Comment:
graph.getStartNode would be useful? Would be nice if we don't have to set or get `Graph.Start` with a variable alltogether but have some funcs for that

Reply:
Will introduce